src/Library/demos/Boxed lists/main.blp

```diff
@@ -0,0 +1,90 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Boxed lists";
+
+  Box {
+    orientation: vertical;
+    spacing: 18;
+    halign: center;
+
+      Adw.PreferencesGroup  {
+        title: "Rows can be groupped";
+        Adw.ActionRow {
+          title: "Action row can have a prefix child";
+
+          [prefix]
+          CheckButton checkbox {
+            active: true;
+          }
+        }
+
+      Adw.ActionRow {
+        title: "Action row can have a suffix child";
+        subtitle: "Select checkbox above to show spinner";
+
+        [suffix]
+        Spinner {
+          spinning: bind checkbox.active;
+        }
+      }
+
+      Adw.EntryRow {
+         title: "A row can be an entry";
+      }
+    }
+
+    Adw.PreferencesGroup {
+      title: "Groups can have a suffix";
+
+      [header-suffix]
+      Adw.ButtonContent {
+        label: "Settings";
+        icon-name: "settings-symbolic";
+      }
+
+      Adw.ComboRow {
+        title: "Choose an item";
+        subtitle: "ComboRow can have a drop down list";
+        model: StringList list {
+          strings ["this", "is", "a", "string", "list"]
+        };
+      }
+
+      Adw.ExpanderRow {
+        title: "Rows can be expandable";
+
+        // remove the line below if you don't want a switch
+        show-enable-switch: true;
+
+        Adw.ActionRow {
+          title: "First row";
+        }
+        Adw.ActionRow {
+          title: "Second row";
+        }
+      }
+
+      Adw.ActionRow {
+        title: "Row can have an interactive button";
+
+        [suffix]
+        Button button {
+          label: "click";
+          valign: center;
+        }
```

Comment:
I've added this new row with a button that changes on click. I can also remove it and re-use in another demo

Reply:
please let me know when this is ready for review

![image](https://user-images.githubusercontent.com/19673/227387570-183382a5-0dc2-4033-889f-c0808120906a.png)